src/SuperVault.sol

```diff
@@ -67,23 +64,23 @@ contract SuperVault is BaseStrategy, ISuperVault {
 
     /// @param superRegistry_ Address of the SuperRegistry contract
     /// @param asset_ Address of the asset token
-    /// @param refundsReceiver_ Address to receive refunds
     /// @param name_ Name of the strategy
     /// @param depositLimit_ Maximum deposit limit
     /// @param superformIds_ Array of Superform IDs
     /// @param startingWeights_ Array of starting weights for each Superform
     constructor(
         address superRegistry_,
         address asset_,
-        address refundsReceiver_,
         string memory name_,
         uint256 depositLimit_,
         uint256[] memory superformIds_,
         uint256[] memory startingWeights_
     )
         BaseStrategy(asset_, name_)
-    {
-        if (superRegistry_ == address(0) || refundsReceiver_ == address(0)) {
+    {   
+        require(superformIds_.length > 0, "EMPTY_SUPERFORM_IDS");
```

Comment:
in superform we have by convention to use "If -> revert" statements with explicit errors